Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/I2CClockRate.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Meadow.Foundation.ICs.IOExpanders
+{
+    public partial class Ft232h
+    {
+        public enum I2CClockRate
```

Comment:
Not specific to this, but I think that `II2cController` should probably have a `SupportedClockRates` or something.